tket/src/serialize/pytket/decoder/wires.rs

```diff
@@ -875,6 +983,25 @@ impl WireTracker {
     pub(super) fn register_unused_parameter_input(&mut self, loaded: LoadedParameter) {
         self.unused_parameter_inputs.push_back(loaded);
     }
+
+    /// Returns a tracked unsupported wire by its [`EncodedEdgeID`].
+    ///
+    /// These are **not** associated with pytket registers or parameters, and
+    /// are used to track connections between unsupported subgraphs and HUGR
+    /// input/output nodes that got encoded as opaque barriers in the pytket
+    /// circuit.
+    ///
+    /// See [`EncodedEdgeID`].
+    pub fn get_unsupported_wire(&self, id: EncodedEdgeID) -> Option<&Wire> {
+        self.unsupported_wires.get(&id)
+    }
+
+    /// Register a new unsupported wire.
```

Comment:
```suggestion
    /// Starts tracking a new unsupported wire.
```
might say that this happens when you traverse the outport, and then `get_unsupported_wire` when you traverse an inport that needs to be connected